docs/embedding/sdk/upgrade.md

```diff
@@ -0,0 +1,71 @@
+---
+title: Upgrading Metabase and the Embedded analytics SDK
+---
+
+# Upgrading Metabase and the Embedded analytics SDK
+
+Here's a basic overview of the steps you'll want to take when upgrading your SDK.
+
+## 1. Read the release post and changelog for both Metabase and the Embedded analytics SDK
+
+- [Release posts](https://www.metabase.com/releases). Gives a good overview of what's in the release, and calls out breaking changes (which are rare).
+- [Metabase changelogs](https://www.metabase.com/changelog) lists Metabase changes.
+- [Embedded analytics SDK changelogs](https://www.metabase.com/changelog/55) lists changes specific to the SDK package.
+
+Check for any relevant changes, especially breaking changes that require you to update your application's code. If there are breaking changes, we'll have docs that'll walk you through what changes you'll need to make and why.
+
+## 2. Test the upgrade
+
+When upgrading to a new major version, you'll want to upgrade both your Metabase and your SDK version in parallel in your staging environment, as having Metabase and the SDK major versions out of sync can cause errors.
```

Comment:
I think testing locally in dev is a requirement and testing in staging is recommended.

* first there may be breaking changes that require updating the code, this needs to be done locally before pushing to staging
* once the code is ready you might want to deploy the changes to stanging for broader testing